fmralign/tests/test_alignment_methods.py

```diff
@@ -194,7 +217,67 @@ def test_ott_backend():
     algo = OptimalTransportAlignment(
         reg=epsilon, metric="euclidean", tol=1e-5, max_iter=10000
     )
-    old_implem = POTAlignment(reg=epsilon, metric="euclidean", tol=1e-5, max_iter=10000)
+    old_implem = POTAlignment(
+        reg=epsilon, metric="euclidean", tol=1e-5, max_iter=10000
+    )
     algo.fit(X, Y)
     old_implem.fit(X, Y)
     assert_array_almost_equal(algo.R, old_implem.R, decimal=3)
+
+
+def test_searchlight_alignment_with_ridge():
+    n_voxels = 99
+    n_time_points = 149
+    n_searchlights = 92
+    n_subjects = 5
+
+    radius = 20
+    searchlights, dists = generate_dummy_searchlights(
+        n_searchlights=n_searchlights,
+        n_v=n_voxels,
+        radius=radius,
+        seed=0,
+    )
+
+    X_train, X_test, _, _, _ = generate_dummy_signal(
+        n_t=n_time_points, n_v=n_voxels, n_s=n_subjects
+    )
+
+    model = INT()
+    model.fit(X_train, searchlights, dists, radius=radius)
+    X_pred = model.transform(X_test)
+
+    # assert that the saved cached files exist
+    b = os.path.exists("cache/")
+    shutil.rmtree("cache")
+    assert b
+    assert X_pred.shape == X_test.shape
+
```

Comment:
The test is a bit weak. It does not assess whether the code works properly. Can you test the output, not only its shape ?

Reply:
What do you mean ? The test is here just to check that there a no errors on a complete run of the code, and that it returns an output of the type and shape we want, nothing more. What would you want me to add ? (all other tests here do the same thing, we are not doing benchmarks in any of these)

Reply:
The tests should check that the output is correct (up to numerical variability, but this can be tamed by seeding)